webcompat/webhooks/helpers.py

```diff
@@ -149,6 +153,23 @@ def get_issue_info(payload):
             'domain': title.partition(' ')[0]}
 
 
+def get_issue_labels(issue_body):
+    """Extract the list of labels from an issue body to be sent to GitHub."""
+    metadata_dict = extract_metadata(issue_body)
+    browser_label = extract_browser_label(metadata_dict)
+    extra_labels = extract_extra_labels(metadata_dict)
+    priority_label = extract_priority_label(issue_body)
+    labelslist = []
+    print(browser_label, extra_labels, priority_label)
+    for label in (browser_label, extra_labels, priority_label):
```

Comment:
I can imagine @karlcow coming up with a sweet list comprehension here... but with the conditionals, a plain ol' for loop seems more readable (to me, anyways).

Reply:
After modifying the test for testing lists.

```
→ nosetests -v --nocapture tests/test_webhook.py
Statuses Initialization…
Fetching milestones from Github…
Milestones saved in data/
Milestones in memory
Extract browser label name. ... ok
Extract 'extra' label. ... ok
test_extract_metadata (tests.test_webhook.TestWebhook) ... ok
Extract priority label. ... ok
POST without bogus signature on labeler webhook is forbidden. ... ok
POST with event not being 'issues' or 'ping' fails. ... ok
POST without signature on labeler webhook is forbidden. ... ok
POST with an unknown action fails. ... ok
GET is forbidden on labeler webhook. ... ok
Extract the right information from an issue. ... ok
Extract list of labels from an issue body. ... ok
Validation tests for GitHub Webhooks. ... ok
POST with PING events just return a 200 and contains pong. ... ok

----------------------------------------------------------------------
Ran 13 tests in 0.073s

OK
```

Reply:
Oh... much simpler. :)

Reply:
> There is an issue though when extra_labels is None. Here a proposal.

(https://github.com/webcompat/webcompat.com/blob/master/webcompat/webhooks/helpers.py#L76 returns `None` as well, so we have to handle `None` anyways).